File: js/twoSum.js
const twoSum = function (numbers, target) {
  let left = 0;
  let right = numbers.length - 1;
  while (numbers[left] + numbers[right] !== target) {
    if (left > right) {
      return -1;
    }
    if (numbers[left] + numbers[right] > target) {
      right--;
    } else {
      left++;
    }
  }
  left++;
  right++;
  return [left, right];
};

File: python/Two Pointers/two_sum.py
def twoSum(numbers: list[int], target: int) -> list[int]:
        left, right = 0, len(numbers) - 1
        while numbers[left] + numbers[right] != target:
            if left > right:
                return -1
            if numbers[left] + numbers[right] > target:
                right -= 1
            else:
                left += 1
        left += 1
        right += 1
        return[left, right]
